src/DacpacTool/BuildOptions.cs

```diff
@@ -10,6 +10,7 @@ public class BuildOptions
         public FileInfo Output { get; set; }
         public SqlServerVersion SqlServerVersion { get; set; }
         public FileInfo[] Input { get; set; }
+        public string InputFile { get; set; }
```

Comment:
No one should be using this directly, so maybe just remove?

Reply:
I think you mean this comment to be on the line above, `public FileInfo[] Input`, right? I'm fine removing if you guys both agree, I just personally prefer to err on the side of not removing previously-working functionality.

Reply:
I prefer to avoid the cost of keeping this old deprecated code alive.

Reply:
I think I'm with @ErikEJ on this. Rather not maintain code that isn't being used anymore. I'll try to give this a review this evening as well.

Reply:
Okay, no problem, I've pushed a change to remove the deprecated code.